### PR TITLE
new icon: remix (original, original-wordmark, line, line-wordmark)

### PR DESCRIPTION
**Pull request by AnshSinghSonkhia**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format new icon: Icon name (versions separated by comma). More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the develop branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the devicon.json file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes** #2127 

# Link to prove your SVG is correct and up-to-date.
- https://remix.run/
- https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c

### credits
Thanks to @weh

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -8834,6 +8834,32 @@
...
+            "remix.run"
+        ],
+        "tags": [
+            "framework",
+            "fullstack",
+            "web"
+        ],
+        "versions": {
+            "svg": [
+                "plain",
```
Comment (canaleal) [inline devicon.json:8849]: Instead of **plain** and **plain-wordmark** use original and **original-wordmark**

**Review by weh — CHANGES_REQUESTED** [review]
hi @AnshSinghSonkhia, thx for your contribution.
You need to check your SVG files according to https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards

- The plain and line versions (with or without wordmark) need to stay as simple as possible. They must have only one color and the paths are united. The color will be removed when being turned into icons so the .svg can have any color. 
- The SVG should be on a single line, with a following empty line
- Check the Line version, to be a path, not strokes

**Comment by AnshSinghSonkhia** [comment]
I made some changes @weh.
Kindly, guide me on any additional changes that you may need.

File: devicon.json [hunk 2]
```diff
@@ -8834,6 +8834,36 @@
...
+                "plain-wordmark",
+                "line",
+                "line-wordmark",
+            ],
+        },
+        "color": "#000000",
+        "aliases": [
```
Comment (canaleal) [inline devicon.json:8856]: Flip these aliases around. 
The base should be `original` and the alias should be `plain`. Do the same for the wordmark version.

**Review by canaleal — CHANGES_REQUESTED** [review]

File: icons/remix/remix-plain.svg
[the remark this answers is not in the document]
Comment (AnshSinghSonkhia) [reply icons/remix/remix-plain.svg:1]: Hey @canaleal, which tools do you use for editing the SVGs?

File: devicon.json [hunk 3]
```diff
@@ -8834,6 +8834,36 @@
...
+        "name": "remix",
+        "altnames": [
+            "remix.run"
+        ],
+        "tags": [
+            "framework",
+            "fullstack",
+            "web"
+        ],
+        "versions": {
```
[the remark this answers is not in the document]
Comment (AnshSinghSonkhia) [reply devicon.json:8847]: Done

File: icons/remix/remix-plain.svg
[the remark this answers is not in the document]
Comment (canaleal) [reply icons/remix/remix-plain.svg:1]: I use a tool called [Inkscape](https://inkscape.org/). It is easy to use and provides utility functions such as resizing, center alignment, etc.

[the remark this answers is not in the document]
Comment (Snailedlt) [reply icons/remix/remix-plain.svg:1]: Worth mentioning that we have a lot of guides for how to do common things in the comments of this issue: #1306 
Most of them are for Inkscape :)

We also have some [recommended resources and tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools) in our wiki

**Review by canaleal — CHANGES_REQUESTED** [review]
Nice, almost there. 👍 
Now we gotta make sure to update the SVGs as they don't meet the standards layed out in the [wiki](https://github.com/devicons/devicon/wiki/SVG-Standards). 

Here's a [tutorial](https://github.com/devicons/devicon/pull/1926#issuecomment-1776317249) on how to update the SVGs, and optimize them.

**Comment by AnshSinghSonkhia** [comment]
> Nice, almost there. 👍 Now we gotta make sure to update the SVGs as they don't meet the standards layed out in the [wiki](https://github.com/devicons/devicon/wiki/SVG-Standards).
> 
> Here's a [tutorial](https://github.com/devicons/devicon/pull/1926#issuecomment-1776317249) on how to update the SVGs, and optimize them.

Thanks for sharing the tutorial @canaleal !!
I have updated the SVGs according to the standards!

**Review by canaleal — CHANGES_REQUESTED** [review]
Excellent work 👍 Almost there.

Don't forget to that all the SVGs need to have 1 path only and must be optimized. 

You can follow the [tutorial](https://github.com/devicons/devicon/pull/1926#issuecomment-1776317249) I provided before on how to unify the paths and optimize the SVGs. The section is called `Unify the Paths and create Plain Icons (Inkscape)`. You don't need to create plain icons, but make sure that the original svgs and the line svgs only have 1 path and 1 fill color. 

Dont forget to optimize all the SVGs too using SVGOMG. You can also find this step in the tutorial.

**Comment by AnshSinghSonkhia** [comment]
> Optimizing the SVG using SVGOMG

# Optimizing is a 1 Click task with SVGVIEWER

I found it more easier to do with it.

1. Visit [https://www.svgviewer.dev](https://www.svgviewer.dev)
2. Upload SVG or paste code

beforeSVG

4. Click on `Prettify` Button.
5. Click on `Optimize` Button.

**BOOM !!!** The SVG is optimized, **without ruining the viewbox**.


optimizedSVG

**Comment by canaleal** [comment]
SVG Viewer is an awesome alternative, a previous contributor even [recommended it](https://github.com/devicons/devicon/issues/1279#issuecomment-1195438321), but notice how the SVGs still have the height, width and even stroke attributes after optimizing them in svg viewer. 

`
<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path style="stroke-width:.270613;fill:#121212;fill-opacity:1" d="M8.389 0v26.09h55.445c14.656 0 21.986 6.29 21.986 16.67 0 11.804-7.33 16.263-21.986 16.263H8.389v26.663h53.818c11.603 0 17.912 3.255 18.93 17.91.77 10.01.674 14.965.582 19.718-.03 1.53-.06 3.038-.06 4.686h34.218c0-7.567 0-14.5-1.15-29.29-1.425-14.857-9.161-22.386-22.393-25.032 15.674-2.036 27.277-14.248 27.277-32.973C119.611 15.671 103.122 0 69.941 0zm0 108.125V128h43.535v-12.72c0-2.674-1.312-7.155-7.356-7.155z"/></svg>
`

For example, I just copied the original svg code from above into SVGOMG and this is how it turned out. The viewbox is still `0 0 128 128`.

`
<svg xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 128 128"><path fill="#121212" d="M8.389 0v26.09h55.445c14.656 0 21.986 6.29 21.986 16.67 0 11.804-7.33 16.263-21.986 16.263H8.389v26.663h53.818c11.603 0 17.912 3.255 18.93 17.91.77 10.01.674 14.965.582 19.718-.03 1.53-.06 3.038-.06 4.686h34.218c0-7.567 0-14.5-1.15-29.29-1.425-14.857-9.161-22.386-22.393-25.032 15.674-2.036 27.277-14.248 27.277-32.973C119.611 15.671 103.122 0 69.941 0zm0 108.125V128h43.535v-12.72c0-2.674-1.312-7.155-7.356-7.155z"/></svg>
`

**Comment by AnshSinghSonkhia** [comment]
> SVG Viewer is an awesome alternative, a previous contributor even [recommended it](https://github.com/devicons/devicon/issues/1279#issuecomment-1195438321), but notice how the SVGs still have the height, width and even stroke attributes after optimizing them in svg viewer.
> 
> `<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path style="stroke-width:.270613;fill:#121212;fill-opacity:1" d="M8.389 0v26.09h55.445c14.656 0 21.986 6.29 21.986 16.67 0 11.804-7.33 16.263-21.986 16.263H8.389v26.663h53.818c11.603 0 17.912 3.255 18.93 17.91.77 10.01.674 14.965.582 19.718-.03 1.53-.06 3.038-.06 4.686h34.218c0-7.567 0-14.5-1.15-29.29-1.425-14.857-9.161-22.386-22.393-25.032 15.674-2.036 27.277-14.248 27.277-32.973C119.611 15.671 103.122 0 69.941 0zm0 108.125V128h43.535v-12.72c0-2.674-1.312-7.155-7.356-7.155z"/></svg>`
> 
> And this is how it looks when I copied the original svg code from above into SVGOMG. The viewbox is still `0 0 128 128`.
> 
> `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path fill="#121212" d="M8.389 0v26.09h55.445c14.656 0 21.986 6.29 21.986 16.67 0 11.804-7.33 16.263-21.986 16.263H8.389v26.663h53.818c11.603 0 17.912 3.255 18.93 17.91.77 10.01.674 14.965.582 19.718-.03 1.53-.06 3.038-.06 4.686h34.218c0-7.567 0-14.5-1.15-29.29-1.425-14.857-9.161-22.386-22.393-25.032 15.674-2.036 27.277-14.248 27.277-32.973C119.611 15.671 103.122 0 69.941 0zm0 108.125V128h43.535v-12.72c0-2.674-1.312-7.155-7.356-7.155z"/></svg>`

Okay, so what do you want me to do now, for the best results?

**Comment by AnshSinghSonkhia** [comment]
# With SVGOMG

```svg
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 128 128"><path fill="#121212" d="M8.389 0v26.09h55.445c14.656 0 21.986 6.29 21.986 16.67 0 11.804-7.33 16.263-21.986 16.263H8.389v26.663h53.818c11.603 0 17.912 3.255 18.93 17.91.77 10.01.674 14.965.582 19.718-.03 1.53-.06 3.038-.06 4.686h34.218c0-7.567 0-14.5-1.15-29.29-1.425-14.857-9.161-22.386-22.393-25.032 15.674-2.036 27.277-14.248 27.277-32.973C119.611 15.671 103.122 0 69.941 0zm0 108.125V128h43.535v-12.72c0-2.674-1.312-7.155-7.356-7.155z"/></svg>
```

Screenshot 2024-03-02 201942

I have not committed these changes, @canaleal 
## It is showing 150px height & width, Will it work?

**Comment by canaleal** [comment]
Yes, I just check the SVG code you provided in the comment above and it looks good on inkscape and icomoon. 

image

image

**Comment by AnshSinghSonkhia** [comment]
Hey @canaleal & @weh,
The required changes are **DONE**. It's good to go, hopefully!!

I Got to learn a lot with this PR. Let me know if we need any more changes.

**Review by canaleal — DISMISSED** [review]
Excellent work! 🚀 
SVGs are good and properly optimized. 

The font icons look good on icomoon, and there are no artifacts. 

image

**Review by canaleal — APPROVED** [review]

**Comment by AnshSinghSonkhia** [comment]
Still, this pr has one pending review @canaleal

**Comment by canaleal** [comment]
Yup, thats pretty much the process. At least 2 reviewers are need to ensure that the changes are correct.

**Review by weh — APPROVED** [review]
LGTM :+1:

**Pull request merged**